confignode/src/main/java/org/apache/iotdb/confignode/persistence/pipe/PipeTaskInfo.java

```diff
@@ -51,6 +51,12 @@ public PipeTaskInfo() {
     this.pipeMetaKeeper = new PipeMetaKeeper();
   }
 
+  /////////////////////////////// Getter ///////////////////////////////
+
+  public PipeMetaKeeper getPipeMetaKeeper() {
```

Comment:
It's a very bad idea to make the meta public accessible. If the PipeMetaKeeper can be accessed and modified outside the PipeTaskInfo.java, the PipeTaskInfo will be meaningless. I strongly recommend that we add some methods in PipeTaskInfo to modify the meta.